cmd/server-main.go

```diff
@@ -591,8 +591,8 @@ func serverMain(ctx *cli.Context) {
 		}
 	}
 
-	if globalCacheConfig.Enabled {
-		// initialize the new disk cache objects.
+	// initialize the new disk cache objects.
+	if globalCacheConfig.Enabled && !globalIsErasure {
```

Comment:
Let's complain about it here as a "WARNING" instead of removing it completely for now and lets take time to purge it.

Add deprecation notice to migrate to "gateway" based setups if disk cache is needed for both single drive and multi-drive.